### DISCUSSION TITLE
HW1 part b

### DISCUSSION THREAD

**Discussion by Amir-M-Vahedi** [cooperrc/advanced-dynamics#22] [Logistics] [answered]
Hi,

In this part, it asks us to describe the horizontal position of the pendulum in both X1 (inertial frame) and X2 (moving frame).
To describe the mentioned variables we need to have the value of $s_1\$.
What is the (initial) value of this parameter?

<img width="658" alt="image" src="https://github.com/cooperrc/advanced-dynamics/assets/115154998/02553ebb-9d06-4a96-9a17-8e28f0f1635f">

**Comment by travismalave** [comment]
I had the same question and then reviewed the Wells reference and realized since we have ax we can back calculate what s1 is in terms of the given velocity and acceleration. Take a look at example 1.1 in the Wells reference. This cleared it up for me.

**Reply by Amir-M-Vahedi** [reply]
You are right and I did the same, but we need the initial value for s1 as we have it for s2 in the homework.

**Reply by travismalave** [reply]
Ah, I misread you question - my bad.  I understand your point. If the figure is to some visual scale, then yes I would agree that there could be some s1_0 value. I think s2 was given since it is a constant value since there is no velocity or acceleration in the y-direction - like the frame is on a track. We may need to assume the initial position of s1_0 = 0 and continue with the relationship of s1 with vx and ax - that is currently what I'm doing. Should be an easy update to the code if we get this value later.

**Reply by cooperrc** [reply, marked as answer]
You can use $s_1(0) = 0$.
